xonsh/completers/bash.py

```diff
@@ -25,6 +25,10 @@
 CACHED_FUNCS = None
 CACHED_FILES = None
 
+BASH_COMPLETE_SCRIPT_PRE = """BASH_COMPLETION_DIR="nonexist"
+BASH_COMPLETION_COMPAT_DIR="nonexist"
+"""
```

Comment:
I think you can just add these to the `BASH_COMPLETE_SCRIPT` below.  No nee to have a 'pre' that is just going to be directly glued in anyways.